vars/publishDistributionBuildResults.groovy

```diff
@@ -37,18 +37,16 @@ void call(Map args = [:]) {
     def inputManifest = readYaml(file: args.inputManifestPath)
     def version = inputManifest.build.version
     def finalJsonDoc = ""
-    def failedComponents = extractUniqueComponents(failureMessages, /(?<=\bError building\s)\S+/)
-    def passedComponents = extractUniqueComponents(passMessages, /(?<=\bSuccessfully built\s)\S+/)
+    List<String> failedComponents = extractComponents(failureMessages, /(?<=\bError building\s).*/, 0)
+    List<String> passedComponents = extractComponents(passMessages, /(?<=\bSuccessfully built\s).*/, 0)
```

Comment:
Hey @prudhvigodithi Can you please add an example output list[] returned by the extract functions in prev and the latest implementation